File: Beginner/Python/1035.py
a,b,c,d = input().split()
a = int(a)
b = int(b)
c = int(c)
d = int(d)
valido = 0
if b > c:
    valido += 1
if d > a:
    valido += 1
if c+d > a+b:
    valido += 1
if c>=0 and d>=0:
    valido += 1
if a%2 == 0:
    valido += 1
if valido >= 5:
    print('Valores aceitos')
else:
    print('Valores nao aceitos')

File: Beginner/C/1035.c
#include <stdio.h>

int main(){

    int a,b,c,d;
    scanf("%i %i %i %i", &a,&b,&c,&d);

    int aceito = 0;

    if(b>c){
        if(d>a){
            if((c+d)>(a+b)){
                if(c>0){
                    if(d>0){
                        if(a%2 == 0){
                            aceito = 1;
                        }
                    }
                }
            }
        }
    }

    if(aceito == 1){
        printf("Valores aceitos\n");
    }else{
        printf("Valores nao aceitos\n");
    }

    return 0;
}